src/lib.rs

```diff
@@ -1,6 +1,87 @@
 #![deny(missing_docs)]
```

Comment:
Will we be able to get rid of this eventually?

Reply:
No, it's so if we ever add a new public facing type, const, function, or module to `ethcontract-rs` but forget to document it, we get build errors.

Reply:
sorry, I read `allow(missing_docs)`, deny is awesome!

Reply:
Denied!